apps/src/code-studio/initApp/clientApi.js

```diff
@@ -131,7 +131,7 @@ var base = {
       })
       .fail(function(request, status, error) {
         var err = new Error('status: ' + status + '; error: ' + error);
-        callback(err, undefined);
+        callback(err, undefined, undefined, request);
```

Comment:
Can you add a comment explaining what these arguments are/ why we're passing undefined?

Reply:
👍